OpenSim/Common/Component.h

```diff
@@ -1434,14 +1434,18 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
     // End of Model Component State Accessors.
     //@} 
 
-    /** @name Dump debugging information to the console */
+    /** @name Print debugging information to the console */
```

Comment:
+1 for changing "dump". Based on a conversation I had with @aseth1, it might be nice to no longer think of these as for debugging. Could we change this to "Print information about this component and its subcomponents to the console."?